capability/capability_linux.go

```diff
@@ -117,6 +117,14 @@ func newPid(pid int) (c Capabilities, retErr error) {
 	return
 }
 
+func ignoreEINVAL(err error) error {
+	// Ignore EINVAL since the op or the capability may not be supported in this system.
```

Comment:
Can you please remove this comment? Instead, explain why we're ignoring EINVAL when we call this function.